labgrid/target.py

```diff
@@ -147,6 +147,8 @@ def get_driver(self, cls, *, name=None, activate=True):
         name -- optional name to use as a filter
         activate -- activate the driver (default True)
         """
+        assert activate == active
```

Comment:
This breaks the default behavior (search for drivers in any state and activate them).

Reply:
Yes I encountered that!

Reply:
Yes, that should be !=...